draft-ietf-quic-transport.md

```diff
@@ -367,6 +367,10 @@ version negotiation and 1-RTT keys are established.
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
 |                          Payload (*)                        ...
 +-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
+|                                                               |
++                        [FNV Hash(64)]                         +
+|                                                               |
++-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+-+
```

Comment:
@kazuho Move to section 6 ? ( Frames and Frame Types)

@mikkelfj may be rename hash or integrity check (like section 5.4)